eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Utxow.hs

```diff
@@ -269,8 +269,8 @@ initialLedgerStateUTXOW ::
   ) =>
   InitialRule (UTXOW era)
 initialLedgerStateUTXOW = do
-  IRC (UtxoEnv slots pp stakepools genDelegs) <- judgmentContext
-  trans @(Core.EraRule "UTXO" era) $ IRC (UtxoEnv slots pp stakepools genDelegs)
+  IRC (UtxoEnv slots pp stakepools genDelegs stakeDistr) <- judgmentContext
```

Comment:
isn't the last value in `UTxOEnv` a `DPState`? either way, I don't think we need it.

Reply:
Its gone so the name doesn't matter any more. I think it harks back to the original branch, which we changed about 10 times, but since the UtxoEnv had the same number of fields, and the only use we made of that variable, was to pass it to another call to the constructor UtxoEnv, the name didn't matter.